interface/batchcom/batchcom.php

```diff
@@ -24,280 +29,217 @@
-            <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
-        });
-    });
-    </script>
-
-	</head>
-	<body class="body_top">
-	<!-- larry's sms/email notification -->
-	<span class="title"><?php include_once("batch_navigation.php");?></span>
-	<!--- end of larry's insert -->
-	<span class="title"><?php xl('Batch Communication Tool','e')?></span>
-	<br><br>
-	<div class="text">
-        <?php
-            echo (xl('No results found, please try again.','','<br>'));
-        ?> </div></body></html> <?php
-        //if results
+            $form_err = xl('No results found, please try again.');
         } else {
             switch ($_POST['process_type']):
-                case $choices[0]: // CSV File
-                    require_once ('batchCSV.php');
-                break;
-                case $choices[1]: // Email
-                    require_once ('batchEmail.php');
-                break;
-                case $choices[2]: // Phone list
-                    require_once ('batchPhoneList.php');
-                break;
+                case $process_choices[0]: // CSV File
+                    generate_csv($res);
+                    exit();
+                case $process_choices[1]: // Email
+                    require_once('batchEmail.php');
+                    exit();
+                case $process_choices[2]: // Phone list
+                    require_once('batchPhoneList.php');
+                    exit();
             endswitch;
         }
-        // end results
 
-        exit ();
     }
 }
 
-//START OUT OUR PAGE....
 ?>
 <html>
 <head>
 <title><?php echo xlt('BatchCom'); ?></title>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<link rel="stylesheet" href="batchcom.css" type="text/css">
-<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.min.css">
-
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-3-1-1/index.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.full.min.js"></script>
-
-<script LANGUAGE="JavaScript">
-$(document).ready(function() {
-    $('.datepicker').datetimepicker({
-        <?php $datetimepicker_timepicker = false; ?>
-        <?php $datetimepicker_showseconds = false; ?>
-        <?php $datetimepicker_formatInput = false; ?>
-        <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
-        <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
-    });
-});
-</script>
-
+<?php Header::setupHeader(['datetime-picker']); ?>
+<style>
+.datepicker {
+    width: 100%;
+}
```

Comment:
if you don't give them a size, I think bootstrap class should make it full width.